### PR TITLE
Add Cloud doc: Export metrics

### LINKED ISSUE #2180: Add how to export your metrics doc in Cloud
### Related code PR

_No response_

### Which part(s) of the docs might be affected or should be updated? And how?

Add how to export your metrics doc in Cloud

### Reference

_No response_

### PR DESCRIPTION
**Pull request by WanYixian**
## Info

- **Description**

  - See [preview](https://pr-2184.d2fbku9n2b6wde.amplifyapp.com/cloud/export-metrics/).

- **Notes**

  - Let me know if any comments, thanks!

- **Related code PR**

  - [ Provide a link to the relevant code PR here, if applicable. ]

- **Related doc issue**
  
  Resolves [ Provide a link to the relevant doc issue here, if applicable. ]

## For reviewers

- **Preview**

  - [ Paste the preview link to the updated page(s) here. Edit this item after the preview site is ready. To find the updated pages, scroll down to locate and open the Amplify preview link and select the **dev** version of the documentation. ]

- **Key points**

  - [ Parts that may need revision or extra consideration. ]

## Before merging

- [ ] I have checked the doc site preview, and the updated parts look good.

- [ ] I have acquired the approval from the owner (and optionally the reviewers) of the code PR and at least one tech writer (`CharlieSYH`, `emile-00`, & `hengm3467`).

### CODE REVIEW

File: cloud/cluster-export-metrics.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
+---
+id: cluster-export-metrics
+title: Export metrics
+description: Export metrics from a RisingWave cloud cluster.
+slug: /export-metrics
+---
+
+This article describes how to use metrics API to export metrics from a RisingWave Cloud cluster to various monitoring systems like Prometheus, DataDog, and InfluxDB. The metrics include all major components such as `etcd`, `frontend`, `compute`, `compactor`, and `meta`, except for those starting with `go.*`.
```
Comment (wjf3121) [inline cloud/cluster-export-metrics.md]: > , except for those starting with `go.*`

I personally feel it's a bit too detailed info for end users. May need insights from doc/product reviewers.

File: cloud/cluster-export-metrics.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+## Step 1: Generate API key
+
+Generate the API key ID and API key secret in the Cloud Portal. See [Generate an API key](organization-service-account.md#generate-an-api-key) for details.
+
+## Step 2: Get Cloud_HOST
+
+Get the corresponding `CLOUD_HOST` for your region and Cloud provider from the table below:
+
+    | Region/CloudProvider | CLOUD_HOST |
```
Comment (hengm3467) [inline cloud/cluster-export-metrics.md]: Somehow the table is presented as a code block.

File: cloud/cluster-export-metrics.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+
+To import the metrics to InfluxDB, you need to configure Telegraf first. See instructions on how to [use Telegraf to scrape Prometheus metrics](https://docs.influxdata.com/influxdb/v2/write-data/developer-tools/scrape-prometheus-metrics/#use-telegraf) and the [Prometheus input plugin](https://github.com/influxdata/telegraf/blob/master/plugins/inputs/prometheus/README.md).
+
+In the Telegraf configuration, you need to fill in the Prometheus input section, see [Prometheus](cluster-export-metrics.md?method=Prometheus#step-3-configure-metric-exporters) for details.
+
+</TabItem>
+
+</Tabs>
+
+## Step 4: Set up dashboard
```
Comment (hengm3467) [inline cloud/cluster-export-metrics.md]:
```suggestion
## Step 4: Set up dashboards
```

File: cloud/cluster-export-metrics.md [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+    | --- | --- |
+    | useast2/aws | canary-useast2-mgmt.risingwave.cloud |
+    | us-east-1/aws | prod-aws-usea1-mgmt.risingwave.cloud |
+    | us-west-2/aws | prod-aws-uswe2-mgmt.risingwave.cloud |
+    | eu-west-2/aws | prod-aws-euwe2-mgmt.risingwave.cloud |
+    | eu-north-1/aws | prod-aws-euno1-mgmt.risingwave.cloud |
+    | us-central1/gcp | prod-gcp-usce1-mgmt.risingwave.cloud |
+    | europe-west3/gcp | prod-gcp-euwe3-mgmt.risingwave.cloud |
+
+## Step 3: Configure metric exporters
```
Comment (hengm3467) [inline cloud/cluster-export-metrics.md]: "metric exporter" is not a proper term here. This step is more about configuring monitoring systems to pull metrics from RisingWave, so we can rename the heading to something like "Configure monitoring systems", and update the description below (the first paragraph) to relect that.

**Review by hengm3467 — APPROVED** [review]
Otherwise looks good to me.

File: cloud/cluster-export-metrics.md (hunk 2 above)
Comment (WanYixian) [reply cloud/cluster-export-metrics.md]: Fixed

File: cloud/cluster-export-metrics.md [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+  - .*
+  auth_type: basic
+  username: {API_KEY}
+  password: {API_SECRET}
+```
+
+#### Notes
+
+- Update the `Cloud_HOST` according to the region and Cloud provider of your RisingWave cluster.
+- Update the `TENANT_CLUSTER_ID` with the specific cluster ID in your RisingWave environment. For example, if your cluster URL is `risingwave-cloud.com/cluster/168/us-central1/overview/`, replace `TENANT_CLUSTER_ID` with the value `168`.
```
Comment (wjf3121) [inline cloud/cluster-export-metrics.md]: nit: tenant is more of a term used internally. Suggest we simply use `CLUSTER_ID` instead.

Comment (WanYixian) [reply cloud/cluster-export-metrics.md]: Thanks! Updated

**Review by wjf3121 — APPROVED** [review]
LGTM! Thanks!

**Pull request merged**